web/packages/plugins/dmt-app/src/index.tsx

```diff
@@ -9,6 +14,7 @@ import Editor from './pages/editor/Editor'
 
 const PageComponent = (props: any) => {
   const { applications, settings } = props
+  // @ts-ignore
```

Comment:
the type error should be fixed in master branch, and not necessary to add the @ts-ignore